src/Table/write_fits.cxx

```diff
@@ -163,6 +163,81 @@ void write_element_given_column_and_row(fitsfile *fits_file, int fits_type,
 /**********************************************************/
 /**********************************************************/
 
+// Labeled_properties are stored as keywords whose values are strings
+// of the form
+
+// label + DOT + XMLATTR_DOT + ATTR_IRSA_VALUE : value  (for prop.value_)
+// label + DOT + XMLATTR_DOT + attrname : attrvalue  (for elements of
+// prop.attributes_)
+
+// where label includes value of prop's ATTR_NAME attribute for
+// uniqueness.
+
+// (We can't yet convert VOTables with more than one RESOURCE to FITS format.
+// 07Dec20)
+
+void write_labeled_properties_as_keywords(
+        fitsfile *fits_file,
+        const std::vector<std::pair<std::string, tablator::Property>>
+                &combined_labeled_properties) {
```

Comment:
Fits header can have one or more header blocks. Each has 2880-bytes.  May be already managed that way somewhere?

Reply:
Are you suggesting that we use multiple header blocks to support VOTables with more than one RESOURCE?  I will look into that if the topic ever arises.

Reply:
No. I knew that this package is for one VO RESOURCE.  Found one fits doc:
https://fits.gsfc.nasa.gov/standard40/fits_standard40aa-le.pdf